MeetupPlatformApi/Features/Authentication/RefreshTokenPair/RefreshTokenPairFeature.cs

```diff
@@ -0,0 +1,62 @@
+﻿namespace MeetupPlatformApi.Features.Authentication.RefreshTokenPair;
+
+using AutoMapper;
+using MeetupPlatformApi.Authentication.Helpers;
+using MeetupPlatformApi.Domain;
+using MeetupPlatformApi.Persistence.Context;
+using MeetupPlatformApi.Seedwork.WebApi;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+[ApiSection(ApiSections.Authentication)]
+public class RefreshTokenPairFeature : FeatureBase
+{
+    private readonly ApplicationContext context;
+    private readonly IMapper mapper;
+    private readonly TokenHelper tokenHelper;
+
+    public RefreshTokenPairFeature(ApplicationContext context, IMapper mapper, TokenHelper tokenHelper)
+    {
+        this.context = context;
+        this.mapper = mapper;
+        this.tokenHelper = tokenHelper;
+    }
+
+    [HttpPost("/api/users/refresh-tokens")]
```

Comment:
Consider adding the `"me"` part to the route (like in the `RevokeUserRefreshTokens` feature).